app/src/main/java/org/mozilla/fenix/settings/SupportUtils.kt

```diff
@@ -37,6 +37,14 @@ object SupportUtils {
     const val GOOGLE_US_URL = "https://www.google.com/webhp?client=firefox-b-1-m&channel=ts"
     const val GOOGLE_XX_URL = "https://www.google.com/webhp?client=firefox-b-m&channel=ts"
 
+    /**
+     * This is a list of search engines that we currently consider to be "General"
+     * search, as opposed to a vertical search engine such as one used for
+     * shopping, book search, etc.
+     */
+    private val GENERAL_SEARCH_ENGINES_IDS =
+        listOf("google", "ddg", "bing", "baidu", "ecosia", "qwant", "yahoo", "yandex")
```

Comment:
When working on this task I looked into how it is implemented on the desktop client. I found a similar approach there and since we do not have a clear delimitation between "general purpose" and "topic specific" search engine types, I tried applying a similar solution.
Here is a link to the desktop list of ids: https://searchfox.org/mozilla-central/source/toolkit/components/search/SearchUtils.sys.mjs#189